packages/neo-one-client/src/__tests__/preconfigured.test.js

```diff
@@ -0,0 +1,81 @@
+/* @flow */
+import * as preconfigured from '../preconfigured';
+import { NEOONEDataProvider, NEOONEProvider } from '../provider';
+import ReadClient from '../ReadClient';
+
+import * as networks from '../networks';
+
+describe('preconfigured', () => {
+  const fakeRPC = 'fakeRPC';
+  const fakeNetwork = 'fakeNet';
+
+  test('provider', () => {
+    const expected = new NEOONEProvider({});
+
+    const result = preconfigured.provider();
+    expect(result.toString()).toEqual(expected.toString());
+  });
+
+  test('mainReadClient - no options', () => {
+    const expected = new ReadClient(
+      new NEOONEDataProvider({
+        network: networks.MAIN,
+        rpcURL: networks.MAIN_URL,
+      }),
+    );
+
+    const result = preconfigured.mainReadClient();
+    expect(result.toString()).toEqual(expected.toString());
+  });
+
+  test('mainReadClient - options', () => {
+    const expected = new ReadClient(
+      new NEOONEDataProvider({
+        network: networks.MAIN,
+        rpcURL: fakeRPC,
+      }),
+    );
+
+    const result = preconfigured.mainReadClient({ rpcURL: fakeRPC });
+    expect(result.toString()).toEqual(expected.toString());
```

Comment:
`toString` doesn't do what you think it does. Try making this test fail by changing `rpcURL` above.